com.unity.cinemachine/Samples~/samples.json

```diff
@@ -1,17 +1,30 @@
 {
   "SharedAssetDependencies": [
```

Comment:
What about adding SharedPackageDependencies?  That would make it less error-prone, since Shared has those dependencies

Reply:
Yes, that's a good idea. I'll add it.